src/worker/api/index.ts

```diff
@@ -2699,6 +2703,134 @@ const updateTeamInfo = async (
 	});
 };
 
+const upsertAwards = async (awards: any, awardsInitial: any): Promise<any> => {
+	awards = awards["aws"];
+	const awardsByPlayer: AwardsByPlayer = [];
+	const simpleAwards =
+		process.env.SPORT === "basketball"
+			? (["mvp", "roy", "smoy", "dpoy", "mip", "finalsMvp"] as const)
+			: (["mvp", "dpoy", "droy", "oroy", "finalsMvp"] as const);
+	const awardNames =
```

Comment:
Since these are also defined exactly the same in doAwards.basketball.ts and doAwards.football.ts, they should be shared. Maybe as a new object in constants.basketball.ts and constants.football.ts which can then be imported both here and in the doAwards files.

Same with `simpleAwards` I think.

Reply:
Actually, a bunch of the code below seems to be duplicated from the doAwards files, so maybe all of this should just be put into a single function which gets called here and in the doAwards files, maybe called `makeAwardsByPlayer(awards)`. I think that would be everything here before the `awardsByPlayerToDelete` stuff.